lib/order.rb

```diff
@@ -0,0 +1,120 @@
+require 'csv'
+require_relative 'customer'
+
+class Order
+  attr_accessor :products, :customer, :fulfillment_status
+  attr_reader :id 
+  
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    if ![:pending, :paid, :processing, :shipped, :complete].include? fulfillment_status
+      raise ArgumentError, 'Invalid fulfillment status.'
```

Comment:
It's helpful to include the invalid status in the error message:

```suggestion
      raise ArgumentError, "Invalid fulfillment status: #{fulfillment_status}"
```